src/roadmap/releasenotes/releasenotes_readme.md

```diff
@@ -0,0 +1,18 @@
+![](img/releasenotes.png)
+
+# ThreeFold Grid Release Notes
+
+ We're delighted to have you here as we explore the latest updates and enhancements to our decentralized grid ecosystem. In these release notes, you'll discover a wealth of information about the exciting features, bug fixes, performance optimizations, and new functionalities that have been introduced in each release. 
+ 
+ Whether you're a developer, a farmer, a user, or simply curious about the cutting-edge advancements happening in the world of distributed computing, these release notes will provide you with valuable insights and keep you up to date with our progress. So dive in, explore the details, and join us in shaping the future of the ThreeFold Grid!
+
+## ThreeFold TFGrid v3.x Release Notes
+- [TFGrid v3.9.0](tfgrid_release_3_9_0)
+- [TFGrid v3.8.0](tfgrid_release_3_8_0)
+- [TFGrid v3.7.0](tfgrid_release_3_7_0)
+- [TFGrid v3.6.1](tfgrid_release_3_6_1)
+- [TFGrid v3.6.0](tfgrid_release_3_6_0)
+- [TFGrid v3.0.0 Alpha-5](tfgrid_release_3_0_a5)
+- [TFGrid v3.0.0 Alpha-4](tfgrid_release_3_0_a4)
+- [TFGrid v3.0.0 Alpha-2](tfgrid_release_3_0_a2)
+- [TFGrid v3.0.0](tfgrid_release_3_0)
```

Comment:
forgot to put .md at the end, will fix

Reply:
fixed

Reply:
@sasha-astiadi These links still didn't work for me after adding the `.md`. i tried other paths but it only worked when i added the releases in the `SUMMARY.md` under the release note section like this.

```
- [ThreeFold Roadmap](roadmap/roadmap_readme.md)
  - [Release Notes](roadmap/releasenotes/releasenotes_readme.md)
    - [TFGrid v3.9.0](roadmap/releasenotes/tfgrid_release_3_9_0.md)
    - [TFGrid v3.8.0](roadmap/releasenotes/tfgrid_release_3_8_0.md)
    - [TFGrid v3.7.0](roadmap/releasenotes/tfgrid_release_3_7_0.md)
    - [TFGrid v3.6.1](roadmap/releasenotes/tfgrid_release_3_6_1.md)
    - [TFGrid v3.6.0](roadmap/releasenotes/tfgrid_release_3_6_0.md)
    - [TFGrid v3.0.0 Alpha-5](roadmap/releasenotes/tfgrid_release_3_0_a5.md)
    - [TFGrid v3.0.0 Alpha-4](roadmap/releasenotes/tfgrid_release_3_0_a4.md)
    - [TFGrid v3.0.0 Alpha-2](roadmap/releasenotes/tfgrid_release_3_0_a2.md)
    - [TFGrid v3.0.0](roadmap/releasenotes/tfgrid_release_3_0.md)
```

I'm not really familiar with mdbook so i'm not sure if this is the correct flow or not. can you check it and verify from your side that this flow is fine and it'll work as intended?

Reply:
There have been many talks on this in the past, see [this](https://github.com/rust-lang/mdBook/issues/702).

As you said @mohamedamer453, if you want some pages to be "rendered"/parsed it needs to be in summary.md.

There could be many "fixes" to this:

1. We either have the different releases .md files in one section, and have a table of contents within the section, much like we have with the [farmer guide](https://www2.manual.grid.tf/TF_Farmer_Guide/TF_Complete_Farmer_Guide/farmer_guide.html)
2. We use [hidden levels ](https://rust-lang.github.io/mdBook/format/configuration/renderers.html#outputhtmlfold)

The hidden levels work great and maybe it's time we start using them, since the ToC is getting pretty dense!

Here are some examples I did locally: 
![Screen Shot 2023-05-25 at 12 44 02 AM](https://github.com/threefoldtech/info_grid/assets/77026219/5f4c3e90-6048-4a6d-aa42-e3c5233da89f)

Then when you click on a section, you get the subsections:
![Screen Shot 2023-05-25 at 12 44 38 AM](https://github.com/threefoldtech/info_grid/assets/77026219/ee23dac1-ccbf-46a0-aac7-abc80563136b)

To achieve this, you only need to change the `book.toml’ file:

```
[output.html.fold]
enable = true    # whether or not to enable section folding
level = 0         # the depth to start folding
```

There might be other ways. So far I found those two.

Let me know what you think @mohamedamer453, @sasha-astiadi  and everyone else.

Reply:
I think that using the hidden levels option is more realistic since having one file for all releases will be too dense. I tried the option locally as you mentioned with the level equal to `0` and it looked much better imo. and i think this would be the best looking option since the docs are not complete yet and the ToC will only get more dense in the future.

![image](https://github.com/threefoldtech/info_grid/assets/101194226/8b105f9f-2c7d-4abd-af30-eb6d8c9fcb95)

another option is to set the level to `1` doesn't look as clean as the first option but it provides more data but it hides nested objects and then the user can expand them.

![image](https://github.com/threefoldtech/info_grid/assets/101194226/4c58ce25-c89d-404c-b30b-0bf94752b0e4)

I think the first option is the best looking but i would like to know what do you think about it @xmonader @sasha-astiadi @Mik-TF

Reply:
> I am curious. There is no book.toml file in there, so how did you set the new parameter output.html.fold?

I was trying it locally so i created a `book.toml` file with the exact config that you mentioned in your comment.

```
[output.html.fold]
enable = true
level = 0
```
but none of my changes are pushed on this branch, i was just trying different variations and didn't want to finalize anything without getting everyone's opinion since it's a major UI change

Reply:
@mohamedamer453  I agree.

Once we get @sasha-astiadi's input, you could do the necessary changes, then we could merge the PR.